parlai/core/torch_ranker_agent.py

```diff
@@ -65,6 +65,19 @@ def add_cmdline_args(cls, argparser):
             '--ignore-bad-candidates', type='bool', default=False,
             help='Ignore examples for which the label is not present in the '
                  'label candidates. Default behavior results in RuntimeError. ')
+        agent.add_argument(
+            '--embedding-normalization', type='bool', default=False,
+            help='normalization after embeddings for transformer. True or False'
+        )
+        agent.add_argument(
+            '--gelu-activation', type='bool', default=False,
+            help='if True use gelu instead of Relu in the FFN. is False use relu'
+        )
+        agent.add_argument(
+            '--reduction-type', type=str, default=None,
```

Comment:
seems like we should have mean be the default, not None. Additionally, None is not a choice that a user can actually pass in via the command line.

Reply:
I put this because maybe a user doesn't want to have any reduction ie --reduction = False, and --reduction is None ? you see what I mean ?

Reply:
If --reduction = False, could we could --reduction-type as None, cf my comment above?

Reply:
Just use 'none' instead of None